src/GrumPHP/Collection/ProcessArgumentsCollection.php

```diff
@@ -65,6 +65,18 @@ public function addOptionalCommaSeparatedArgument($argument, array $values, $del
 
     /**
      * @param string $argument
+     * @param string $value
+     */
+    public function addOptionalSeparatedArgument($argument, $value = null)
```

Comment:
This method was already implemented in the `addOptionalArgumentWithSeparatedValue` method:
https://github.com/jhuet/grumphp/blob/8ff4bb0ff833d00ad0ea010cc164961832173df7/src/GrumPHP/Collection/ProcessArgumentsCollection.php#L42

Can you remove the `addOptionalSeparatedArgument` method and use the one above instead??